js/game.js

```diff
@@ -4,89 +4,135 @@ const cardArray = [
         name: "spider",
         image: "assets/svgs/spider-solid.svg",
         pictureid: 0,
+        matched: false,
     },
     {
         id: 1,
         name: "ghost",
         image: "assets/svgs/ghost-solid.svg",
         pictureid: 1,
+        matched: false,
     },
     {
         id: 2,
         name: "crow",
         image: "assets/svgs/crow-solid.svg",
         pictureid: 2,
+        matched: false,
     },
     {
         id: 3,
         name: "bat",
         image: "assets/svgs/bat.svg",
         pictureid: 3,
+        matched: false,
     },
     {
         id: 4,
         name: "broom",
         image: "assets/svgs/broom-solid.svg",
         pictureid: 4,
+        matched: false,
     },
     {
         id: 5,
         name: "hat",
         image: "assets/svgs/hat-wizard-solid.svg",
         pictureid: 5,
+        matched: false,
     },
     {
         id: 6,
         name: "spider",
         image: "assets/svgs/spider-solid.svg",
         pictureid: 0,
+        matched: false,
     },
     {
         id: 7,
         name: "ghost",
         image: "assets/svgs/ghost-solid.svg",
         pictureid: 1,
+        matched: false,
     },
     {
         id: 8,
         name: "crow",
         image: "assets/svgs/crow-solid.svg",
         pictureid: 2,
+        matched: false,
     },
     {
         id: 9,
         name: "bat",
         image: "assets/svgs/bat.svg",
         pictureid: 3,
+        matched: false,
     },
     {
         id: 10,
         name: "broom",
         image: "assets/svgs/broom-solid.svg",
         pictureid: 4,
+        matched: false,
     },
     {
         id: 11,
         name: "hat",
         image: "assets/svgs/hat-wizard-solid.svg",
         pictureid: 5,
+        matched: false,
     },
 ];
-
 const shuffle = (cards) => {
```

Comment:
Why did you delete the blank line above this function? I think it helps readability.

Same for code below.